axiom/orgs.go

```diff
@@ -15,43 +15,53 @@ type Plan uint8
 
 // All available deployment or organization plans.
 const (
-	Free       Plan = iota + 1 // free
-	Trial                      // trial
-	Pro                        // pro
-	Enterprise                 // enterprise
-	Comped                     // comped
+	emptyPlan Plan = iota //
```

Comment:
No. In some cases (`setup-axiom` action), there is no license returned. This will make sure we can handle this properly without reporting an invalid license tier.